<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilibili直播弹幕姬</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_roomUrl">
       <property name="text">
        <string>直播间地址：http://live.bilibili.com/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QLabel" name="label_statistic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>暂无统计信息</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QCheckBox" name="checkBox_scollerOn">
       <property name="text">
        <string>侧边栏</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <layout class="QGridLayout" name="gridLayout_tab">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_log">
          <attribute name="title">
           <string>日志</string>
          </attribute>
          <widget class="QPlainTextEdit" name="plainTextEdit_logs">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>486</width>
             <height>268</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_plugin">
          <attribute name="title">
           <string>插件</string>
          </attribute>
          <widget class="QTableView" name="tableView_plugins">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>486</width>
             <height>268</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_setting">
          <attribute name="title">
           <string>设置</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>481</width>
             <height>271</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_group">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_mainWin">
              <property name="title">
               <string>弹幕姬设置</string>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>211</width>
                 <height>241</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_win">
                <item row="2" column="0">
                 <widget class="QCheckBox" name="checkBox_noticeOn">
                  <property name="text">
                   <string>最小化后启用通知</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="checkBox_keepTop">
                  <property name="text">
                   <string>本窗口置顶</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox_autoReconn">
                  <property name="text">
                   <string>自动重连</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_scroller">
              <property name="title">
               <string>侧边栏设置</string>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_4">
               <property name="geometry">
                <rect>
                 <x>9</x>
                 <y>20</y>
                 <width>224</width>
                 <height>241</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_scroller">
                <item row="0" column="0" colspan="2">
                 <widget class="QCheckBox" name="checkBox_scrollerResize">
                  <property name="text">
                   <string>显示窗体轮廓（以调整大小位置）</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="pushButton_setFont">
                  <property name="text">
                   <string>字体设置</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pushButton_setColor">
                  <property name="text">
                   <string>颜色设置</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>弹幕停留时长(s)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBox_stayTime">
                  <property name="value">
                   <number>4</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_roomID"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_disconn">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_conn">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
